### PR TITLE
DOC-5722 - added details on configuring ledger recovery cleanup job

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
Preview at:

https://paul.preview.docs.r3.com/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.html

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md [hunk 1]
```diff
@@ -106,6 +107,48 @@ enterpriseConfiguration {
...
+
+## Ledger recovery distribution record cleanup
+
+The Ledger Recovery distribution record cleanup removes distribution records earlier than a certain point in time. For more information on Ledger Recovery and its associated distribution records, see [Ledger Recovery](../../ledger-recovery-flow.md).
+
+The Ledger Recovery distribution record cleanup only runs if one of the following parameters has been configured:
+
+* If the [network parameter](../../network/available-network-parameters.md) 'recoveryMaximumBackupInterval' is defined, then it is used and given first precedence
+* Else, if the node parameter `enterpriseConfiguration.ledgerRecoveryConfiguration.recoveryMaximumBackupInterval` is defined, it is used.
+* Otherwise, it Ledger Recovery distribution record cleanup does not run.
```
Comment (shamsasari) [inline content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md:119]: typo: it

**Review by shamsasari — COMMENTED** [review]
You should mention that entries from the `node_sender_distr_recs` and `node_receiver_distr_recs` tables are removed

File: content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md [hunk 2]
```diff
@@ -106,6 +107,48 @@ enterpriseConfiguration {
 }
 ```
 
+
+## Ledger recovery distribution record cleanup
```
Comment (amiegrace12) [inline content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md:111]:
```suggestion
## Ledger Recovery distribution record cleanup
```

File: content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md [hunk 3]
```diff
@@ -24,6 +24,7 @@ The following maintenance tasks are currently supported:
 
 - Perform RPC Audit table clean-up
 - Run message ID clean-up
+- Ledger recovery distribution record clean-up
```
Comment (amiegrace12) [inline content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md:27]:
```suggestion
- Ledger Recovery distribution record clean-up
```

File: content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md [hunk 4]
```diff
@@ -106,6 +107,48 @@ enterpriseConfiguration {
...
 ```
 
+
+## Ledger recovery distribution record cleanup
+
+The Ledger Recovery distribution record cleanup removes distribution records earlier than a certain point in time. The job deletes entries from the `NODE_SENDER_DISTR_RECS` and `NODE_RECEIVER_DISTR_RECS` tables. For more information on Ledger Recovery and its associated distribution records, see [Ledger Recovery](../../ledger-recovery-flow.md).
+
+The Ledger Recovery distribution record cleanup only runs if one of the following parameters has been configured:
+
+* If the [network parameter](../../network/available-network-parameters.md) 'recoveryMaximumBackupInterval' is defined, then it is used and given first precedence
```
Comment (amiegrace12) [inline content/en/platform/corda/4.11/enterprise/node/operating/maintenance-mode.md:117]:
```suggestion
* If the [network parameter](../../network/available-network-parameters.md) 'recoveryMaximumBackupInterval' is defined, then it is used and given first precedence.
```

**Pull request merged**